storage/aptosdb/src/transaction_store/mod.rs

```diff
@@ -78,16 +80,80 @@ impl TransactionStore {
         ))
     }
 
+    // TODO[Orderless]: Update this so that the user can specify even the range of chain timestamps
```

Comment:
that seems belonging to a separate API
for e.g. split to `_by_versions`, `_by_timestamps` flavors